vmdb/app/presenters/menu/polymorph_item.rb

```diff
@@ -0,0 +1,7 @@
+module Menu
+  class PolymorphItem < Item
+    def name
+      super.call
```

Comment:
Oh wait I see...you are passing a lambda.  But why?  hybrid_name can just return the name directly...then you don't need this class at all.